include/diaspora.php

```diff
@@ -2876,8 +2876,9 @@ public static function build_status($item, $owner) {
 					"created_at" => $created,
 					"provider_display_name" => $item["app"]);
 
-			if (count($location) == 0)
+			if (!isset($location["lat"]) OR !isset($location["lng"])) {
```

Comment:
What do you think about adding a short comment about the fact that diaspora does need properties `lat` and `lng` for the location otherwise it wouldn't work